Resources/public/js/views/fields/ez-binarybase-editview.js

```diff
@@ -116,7 +116,7 @@ YUI.add('ez-binarybase-editview', function (Y) {
             var file = this.get('file'),
                 fieldValue;
 
-            if ( !this.get('updated') ) {
+            if ( !this.get('updated') && this.get('field').fieldValue && (this.get('field').fieldValue.fileName == file.name) ) {
```

Comment:
comparing file names to check if the file has been updated is a bad idea, what if I update the field with a different file but with the same name ?